src/ApiService/ApiService/onefuzzlib/orm/Orm.cs

```diff
@@ -15,6 +15,14 @@ public interface IOrm<T> where T : EntityBase {
         Task<ResultVoid<(int, string)>> Insert(T entity);
         Task<ResultVoid<(int, string)>> Delete(T entity);
 
+        IAsyncEnumerable<T> ListAll();
+        IAsyncEnumerable<T> ListPartition(string partitionKey);
+        IAsyncEnumerable<T> SearchByRowKey(string rowKey);
+        IAsyncEnumerable<T> SearchByTimeRange(DateTimeOffset min, DateTimeOffset max);
```

Comment:
we should have a the same prefix for these. I propose we stick with Search/SearchBy